frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -286,32 +278,11 @@ class ChartSettings extends Component {
       </SectionContainer>
     );
 
-    const widgetList = visibleWidgets.map(widget => (
-      <ChartSettingsWidget
-        key={widget.id}
-        {...widget}
-        {...extraWidgetProps}
-        setSidebarPropsOverride={setSidebarPropsOverride}
-      />
-    ));
-
     const onReset =
       !_.isEqual(settings, {}) && (settings || {}).virtual_card == null // resetting virtual cards wipes the text and broke the UI (metabase#14644)
         ? this.handleResetSettings
         : null;
 
-    // custom render prop layout:
```

Comment:
I couldn't find anywhere that this is actually used (along with the `SidebarPropsOverride` function). Hoping it's safe to remove

Reply:
Just double-checked. I think you're right.